driver/razerkbd_driver.c

```diff
@@ -2921,19 +2950,20 @@ static ssize_t razer_attr_write_matrix_custom_frame(struct device *dev, struct d
             request.transaction_id.id = 0x1F;
             break;
 
-        case USB_DEVICE_ID_RAZER_BLACKWIDOW_V4_PRO:
-            request = razer_chroma_extended_matrix_set_custom_frame(row_id, start_col, stop_col, (unsigned char*)&buf[offset]);
-            request.transaction_id.id = 0x1F;
-            want_response = false;
-            break;
-
```

Comment:
Why did you remove the code above?

Reply:
it's not removed, just moved below to group `USB_DEVICE_ID_RAZER_BLACKWIDOW_V3_PRO_WIRELESS` with others, I assumed V4 should come after V3 ?

Reply:
Looks like rebasing on current master there was movement around the V4 one, things should get back in order?